static/app/views/organizationIntegrations/integrationRepos.spec.jsx

```diff
@@ -5,7 +5,7 @@ import IntegrationRepos from 'sentry/views/organizationIntegrations/integrationR
 
 describe('IntegrationRepos', function () {
   const org = TestStubs.Organization();
-  const integration = TestStubs.GitHubIntegration();
+  const integration = TestStubs.GithubIntegration();
```

Comment:
should we create a separate PR  these name changes?

Reply:
I'll revert and hardcode this - I cannot just rename the file as then we'll have the whole case-insensitive fs problem. The disk file here is `githubIntegration` where it should be `gitHubIntegration` :(